sim/scenarios/helper/quic-network-simulator-helper.cc

```diff
@@ -13,6 +14,13 @@
 
 using namespace ns3;
 
+void onSignal(int signum) {
+  std::cout << "Received signal: " << signum << std::endl;
+  // see https://gitlab.com/nsnam/ns-3-dev/issues/102
+  Simulator::Stop();
```

Comment:
Is this necessary to flush all the pcaps?

Reply:
This is what `NS_FATAL_ERROR` does. Don't ask me why, I don't understand it, but it's what people more familiar with ns3 recommended: https://gitlab.com/nsnam/ns-3-dev/issues/102. And it seems to work.